src/helpers/fallbackauthdialog.js

```diff
@@ -1,35 +1,64 @@
 /*
  * Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. See full license in root of repo. -->
  *
- * This file shows how to use MSAL.js to get an access token to Microsoft Graph an pass it to the task pane.
+ * This file shows how to use MSAL.js to get an access token to your server and pass it to the task pane.
  */
 
-/* global console, localStorage, Office, window */
+/* global console, localStorage, location, Office, window */
 
-import { PublicClientApplication } from "@azure/msal-browser";
+import { LogLevel, PublicClientApplication } from "@azure/msal-browser";
+import { callGetUserData } from "./middle-tier-calls";
```

Comment:
Because of the layers of indirection, it was easy to confuse which function was being referred to.  

1. excel.js calls: sso-helper.getUserData(callback)
2. sso-helper.getUserData(callback) calls: middle-tier-calls.callGetUserData(token)
3. middle-tier-calls.callGetUserData(token) calls: Make rest call to middle tier '/getuserdata'
4. rest middle tier '/getuserdata' routes to: msgraph-helper.getUserData(req, res, next) 
5. msgraph-helper.getUserData(req, res, next) calls: msgraph-helper.getGraphData(accessToken, apiUrl, queryParams)

All of those calls along the chain could be called "getUserData".  Having a different name in the middle helped to distinguish it a little on the office client side, but I do agree this is quirky.  I'm open to suggestions.

Reply:
OK. Sounds good.